Xamarin.Forms.Platform.Android/FastRenderers/FrameRenderer.cs

```diff
@@ -1,56 +1,34 @@
 using System;
-using System.Collections.ObjectModel;
-using System.Collections.Specialized;
 using System.ComponentModel;
 using Android.Content;
-using Android.Support.V4.View;
 using Android.Support.V7.Widget;
 using Android.Views;
 using AColor = Android.Graphics.Color;
 using AView = Android.Views.View;
 
-namespace Xamarin.Forms.Platform.Android.AppCompat
+namespace Xamarin.Forms.Platform.Android.FastRenderers
```

Comment:
Do you mean keep the old namespace? I thought we had to put them in a _different_ namespace so people could still use the old renderers.